great_expectations/rule_based_profiler/helpers/simple_semantic_type_filter.py

```diff
@@ -127,9 +127,23 @@ def _infer_semantic_domain_type_from_table_column_type(
         column_type: str = str(column_types_dict_list[0]["type"]).upper()
 
         semantic_column_type: SemanticDomainTypes
-        if column_type in (
-            {type_name.upper() for type_name in ProfilerTypeMapping.INT_TYPE_NAMES}
-            | {type_name.upper() for type_name in ProfilerTypeMapping.FLOAT_TYPE_NAMES}
+        if (
+            column_type
+            in (
+                {type_name.upper() for type_name in ProfilerTypeMapping.INT_TYPE_NAMES}
+                | {
+                    type_name.upper()
+                    for type_name in ProfilerTypeMapping.FLOAT_TYPE_NAMES
+                }
+            )
+            or any(
```

Comment:
wanting to check my understanding: by using `startswith()` it makes this inferrence more general?

Reply:
@Shinnnyshinshin This is to insure that `DECIMAL(a, b)` can be matched to `DECIMAL` (we cannot match on all possible combinations of precision).  Thanks.